packages/components/resultCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, Box, Avatar, Text } from '@chakra-ui/react';
+
+import type { NextPage } from 'next';
+
+const ResultCard: NextPage = () => {
+  return (
+    <>
+      <Box display="flex" flexDirection="column" width="50vw" px="2rem">
```

Comment:
Let's enhance how this component looks, you should already use the final background color. Remember to use chakra components to beautify your application, like [Badges](https://chakra-ui.com/docs/components/badge) or use text with colors. The card could look more like a card, with box-shadow, a border...